doc/nrf/protocols/wifi/sniffer_rx_operation.rst

```diff
@@ -0,0 +1,138 @@
+﻿.. _ug_nrf70_developing_raw_ieee_80211_packet_reception:
+
+Raw IEEE 802.11 packet reception using Monitor mode
+###################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF70 Series devices support Monitor mode.
+You must configure the nRF70 device in Monitor mode to operate it as an 802.11 wireless packet sniffer.
+
+In this mode, the device listens on a configured channel and receive all IEEE 802.11 traffic, including traffic from all BSSIDs operating on that channel.
+The device then prepends the captured traffic with metadata before passing it to the application layer using raw sockets.
+The raw RX header contains parameters of the received packet, including data rate, signal strength, reception frequency, and packet type, and packet type (whether it is Legacy, HT, VHT, or HE).
+
+.. _ug_nrf70_developing_enabling_802.11_monitor_mode:
+
+Enabling Monitor mode
+*********************
+
+To be able to use the Monitor mode in your application, you need to enable the :kconfig:option:`CONFIG_NRF700X_RAW_DATA_RX` Kconfig option in the project configuration.
+
+.. _ug_nrf70_developing_monitor_mode_operation:
+
+Monitor mode operation
+**********************
+
+Monitor mode can be enabled or disabled on the nRF70 Series device at runtime, using the ``NET_REQUEST_WIFI_MODE`` network management runtime API.
+This runtime API can be used to disable the Monitor mode in the driver, when the packet sniffer operation is no longer required in the user application.
+
+Monitor mode on the nRF70 device cannot operate concurrently with other modes such as Station mode or SoftAP mode.
+
+The user needs to configure the operating Wi-Fi channel on which the nRF70 device will monitor and receive 802.11 packets.
+When the device operates in Monitor mode, all 802.11 packets received on the configured channel will be sent up the stack by the nRF70 Series device and the Wi-Fi driver.
+
+To set the desired channel for 802.11 packet reception, use the ``NET_REQUEST_WIFI_CHANNEL`` runtime NET-management API.
```

Comment:
Should it be: Network management API?
I couldn't find NET-management used in other docs; if it is used elsewhere/is official name of the API then please leave as is.